src/index.js

```diff
@@ -4,6 +4,6 @@ import App from './App';
 import './index.css';
 
 render(
-  <App />,
+  <App url="https://raw.githubusercontent.com/ryanvannin/appUno/master/initialdata.json" pollInterval={2000} />,
```

Comment:
Can you submit the initialdata.json file together with the pr and refer to it instead of relying on the one on the fork?